docs/integration/glossary.md

```diff
@@ -0,0 +1,49 @@
+# DTS Integration Glossary
+
+We use the following terms in the DTS Integration Guide.
+
+* **DTS**: The [BER data transfer system](https://kbase.github.io/dts/)
+* **Frictionless DataPackage**: A [representation](https://specs.frictionlessdata.io/data-package/)
+  for a collection of **Frictionless DataResources**. The DTS generates **transfer
+  manifests** in this format.
+* **Frictionless DataResource**: A [representation](https://specs.frictionlessdata.io/data-resource/)
+  of metadata for an individual resource (file)
+* **Resource**: A file, with a unique identifier, that can be transferred from
+* **Resource endpoint**: an endpoint provided by your database that accepts an
+  HTTP `GET` request with a list of file IDs, and provides a response containing
+  essential informatіon (the file's location, its type and other important
+  metadata) for each of the corresponding files. Read [this](resources.md) for
```

Comment:
```md
See the [resources guide](resources.md) for more detailed information.
```
?

Reply:
I guess I did the curt link because it's a section with a name that doesn't fit easily into this context. I could just strike the link, but I think it still might be useful.